elements/lisk-p2p/src/utils/misc.ts

```diff
@@ -126,10 +124,10 @@ export const getBucket = (options: {
 	readonly secret: number;
```

Comment:
Can we change the name to `getBucketId`. `getBucket`? It's a bit misleading as it only returns a mere number.